2019-11-20 23:16:57.993 - [Info] MITM Version: 1.26
2019-11-20 23:16:58.009 - [Info] Auto Access Enabled: true
2019-11-20 23:16:58.010 - [Debug] [Init] Auto Access Barrier: true
2019-11-20 23:16:58.062 - [Info] SSH man-in-the-middle server for 172.20.0.6 listening on 0.0.0.0:10105
2019-11-20 23:45:53.857 - [Debug] [Connection] Attacker connected: 139.59.3.151 | Client Identification: SSH-2.0-libssh-0.6.3
2019-11-20 23:45:55.318 - [Debug] [Auth] Attacker 139.59.3.151 trying to authenticate with "password"
2019-11-20 23:45:55.320 - [Debug] [Auto Access] Attacker: 139.59.3.151, Threshold: 1, Attempts: 1
2019-11-20 23:45:55.320 - [Debug] [Auto Access] Compromising the honeypot
2019-11-20 23:45:55.320 - [Debug] [Auto Access] Adding the following credentials: "raverdino:raverdino"
2019-11-20 23:45:55.553 - [Debug] [Auth] Valid credentials - Password Authentication
2019-11-20 23:45:55.554 - [Debug] [LXC] Attempting to connect to the honeypot: 172.20.0.6
2019-11-20 23:45:55.603 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2019-11-20 23:45:56.488 - [Debug] [Connection] Attacker connected: 139.59.3.151 | Client Identification: SSH-2.0-libssh-0.6.3
2019-11-20 23:45:57.792 - [Debug] [Auth] Attacker 139.59.3.151 trying to authenticate with "password"
2019-11-20 23:45:57.793 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2019-11-20 23:45:57.794 - [Debug] [Auth] Attacker: 139.59.3.151 has so far made 1 attempts. Remaining: 5 attempts
2019-11-20 23:45:58.238 - [Debug] [Connection] Attacker closed the connection
2019-11-20 23:45:58.706 - [Debug] [EXEC] Noninteractive mode attacker command: cat /proc/cpuinfo | grep name | wc -l
